vcpkg.json

```diff
@@ -0,0 +1,93 @@
+{
+    "name": "freerdp",
+    "version-string": "3.3.0",
+    "description": "A free implementation of the Remote Desktop Protocol (RDP)",
+    "homepage": "https://github.com/FreeRDP/FreeRDP",
+    "license": "Apache-2.0",
+
+    "dependencies": [
+        "zlib",
+        "openssl"
+    ],
+
+    "default-features": [
+    ],
+
+    "features": {
+        "all": {
+            "description": "Enable all features",
+            "dependencies": [
+                {
+                    "name": "ffmpeg",
+                    "features": [
+                        "openh264"
+                    ]
+                },
+                "cjson",
+                "libusb",
+                {
+                    "name":"uriparser",
+                    "platform": "linux"
```

Comment:
where does this idea come from?

Reply:
https://github.com/FreeRDP/FreeRDP/wiki/Compilation#dependencies